From hannes@hanneseder.net  Wed Mar 11 11:39:10 2009
From: Hannes Eder <hannes@hanneseder.net>
Date: Fri, 27 Feb 2009 02:04:31 +0100
Subject: USB: host: fix sparse warning: Using plain integer  as NULL pointer
To: Greg Kroah-Hartman <gregkh@suse.de>
Message-ID: <154e089b0902261704j8dba825q59411db576a157e8@mail.gmail.com>

From: Hannes Eder <hannes@hanneseder.net>

Fix this sparse warning:
 drivers/usb/host/oxu210hp-hcd.c:2687:42: warning: Using plain integer as NULL pointer

Signed-off-by: Hannes Eder <hannes@hanneseder.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/oxu210hp-hcd.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/host/oxu210hp-hcd.c
+++ b/drivers/usb/host/oxu210hp-hcd.c
@@ -2684,7 +2684,7 @@ static int oxu_reset(struct usb_hcd *hcd
 	oxu->urb_len = 0;
 
 	/* FIMXE */
-	hcd->self.controller->dma_mask = 0UL;
+	hcd->self.controller->dma_mask = NULL;
 
 	if (oxu->is_otg) {
 		oxu->caps = hcd->regs + OXU_OTG_CAP_OFFSET;
